ui/components/multichain/account-list-item/account-list-item.types.js

```diff
@@ -0,0 +1,5 @@
+export const MenuOptionTypes = {
```

Comment:
Can this be renamed to `AccountListItemMenuTypes` to be more specific?

Reply:
Yup https://github.com/MetaMask/metamask-extension/pull/23232/commits/f04561a22fb16e2948e6b45218a8857144784531